charts/fulcio/values.schema.json

```diff
@@ -513,6 +513,26 @@
                                 "fileca"
                             ]
                         },
+                        "kms_resource": {
+                            "$id": "#/properties/server/properties/args/properties/kms_resourse",
+                            "type": "string",
+                            "title": "The kms resource schema",
+                            "description": "The KMS URI for backend KMS",
+                            "default": "",
+                            "examples": [
+                                "gcpkms://foo/bar"
+                            ]
+                        },
+                        "kms_cert_chain": {
+                            "$id": "#/properties/server/properties/args/properties/kms_cert_chain",
+                            "type": "string",
+                            "title": "The kms chain schema",
+                            "description": "Certificate chain for KMS CA backend",
```

Comment:
Can we note that the chain starts with the issuing CA and ends with the root CA?